### PR TITLE
Add tags

### PR DESCRIPTION
**Pull request by OniiCoder**

### CODE REVIEW

**Review by iamkarsoft — APPROVED** [review]
awesome work!

File: src/Models/Group.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    protected $table = 'article_groups';
+
+    public static function icon(): string
+    {
+        return Icon::DOCUMENT_TEXT;
+    }
+
+    public function editorShowViewRoute(string $layout = null): string
+    {
+//        return route('lego.blog.group.editor', [
```
Comment (BaffourAdu) [inline src/Models/Group.php]: @OniiCoder forgot to remove comments here

**Review by garos4 — APPROVED** [review]

**Review by BaffourAdu — CHANGES_REQUESTED** [review]

File: src/Models/Group.php (hunk 1 above)
Comment (OniiCoder) [reply src/Models/Group.php]: @BaffourAdu  the `editorShowViewRoute ` function is required to be implemented but I am not using the editor for the model for now at least or maybe there is another way to not allow the function to be required.

File: src/Http/Livewire/Models/TagForm.php [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+use Helix\Lego\Models\Model;
+use Illuminate\Support\Collection;
+
+class TagForm extends Form
+{
+    public Tag $tag;
+    public Collection $selectedArticles;
+    public array $selectedArticlesIds = [];
+
+//    protected $listeners = [
```
Comment (bmmage) [inline src/Http/Livewire/Models/TagForm.php]: Can you remove this commented code?

File: src/Models/Tag.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+    public static function icon(): string
+    {
+        return Icon::DOCUMENT_TEXT;
+    }
+
+    public function editorShowViewRoute(string $layout = null): string
+    {
+//        return route('lego.blog.tag.editor', [
+//            'tag' => $this,
```
Comment (bmmage) [inline src/Models/Tag.php]: I'd seen the discussion between you and BaffourAdu on this. I'd recommend removing the current comment and possibly put a 'unused' type comment just for clarity.

**Review by bmmage — APPROVED** [review]
Looks good, great work! I've added two comments around some code comments 😅 . Throwing an approval on this since those comments are more cosmetic than functional.

**Review by garos4 — APPROVED** [review]

File: src/Models/Tag.php [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+use Helix\Fabrick\Icon;
+use Helix\Lego\Models\Contracts\Sectionable;
+use Helix\Lego\Models\Model as LegoModel;
+use Helix\Lego\Models\Traits\HasSections;
+
+class Tag extends LegoModel implements Sectionable
+{
+    use HasSections;
+
+    protected $table = 'article_tags';
```
Comment (bmmage) [inline src/Models/Tag.php:14]: Would it be worth making the 'tags' more generic and part of strata proper so we could tag other models in the future?

**Review by tonning — APPROVED** [review]

**Pull request merged**